apps/rush-lib/src/RushConstants.ts

```diff
@@ -79,13 +79,21 @@ export namespace RushConstants {
   export const packageJsonFilename: string = 'package.json';
 
   /**
-   * The filename ("pinned-versions.json") for an optional configuration file
-   * that stores a table of dependencies that should be pinned to a specific
-   * version for all projects.  This configuration file should go in the
-   * "common/config/rush" folder.
+   * The filename ("pinned-versions.json") for an old configuration file that
+   * that is no longer supported.
+   *
+   * @deprecated This feature has been superseded by the "preferredVersions" setting
+   * in common-versions.json
    */
   export const pinnedVersionsFilename: string = 'pinned-versions.json';
```

Comment:
>pinnedVersionsFilename [](start = 15, length = 22)

Webpack does a neat thing where it prints a warning when you try to access a deprecated member.